src/graphql/main/mutations.ts

```diff
@@ -1,5 +1,6 @@
 import { GT } from "@graphql/index"
 
+import DeviceNotificatinoTokenCreateMutation from "@graphql/root/mutation/device-notification-token-create"
```

Comment:
Small type here and in a few other places that use this value
```suggestion
import DeviceNotificationTokenCreateMutation from "@graphql/root/mutation/device-notification-token-create"
```